scrapy_poet/injection.py

```diff
@@ -15,14 +15,14 @@
 from scrapy.statscollectors import StatsCollector
 from scrapy.utils.conf import build_component_list
 from scrapy.utils.defer import maybeDeferred_coro
-from scrapy.utils.misc import load_object
+from scrapy.utils.misc import load_object, create_instance
 
 from scrapy_poet.cache import SqlitedictCache
 from scrapy_poet.injection_errors import (UndeclaredProvidedTypeError,
                                           NonCallableProviderError,
                                           InjectionError)
 from scrapy_poet.overrides import OverridesRegistryBase, \
-    PerDomainOverridesRegistry
+    OverridesRegistry
 from scrapy_poet.page_input_providers import PageObjectInputProvider
 from scrapy_poet.api import _CALLBACK_FOR_MARKER, DummyResponse
```

Comment:
I believe it was to discourage from users to use `_CALLBACK_FOR_MARKER` to see if callbacks needs a Scrapy response _(checking if such marker is present)_ since the `is_callback_requiring_scrapy_response()` function already does this.